README.md

```diff
@@ -1,20 +1,41 @@
 <div align="center">
+  <!-- <img src="https://github.com/allenai/OLMo/assets/8812459/774ac485-a535-4768-8f7c-db7be20f5cc3" width="300"/> -->
+  <img src="https://allenai.org/olmo/olmo-7b-animation.gif" alt="OLMo Logo" width="600" style="margin-left:'auto' margin-right:'auto' display:'block'"/>
+  <br>
+  <br>
```

Comment:
I don't love this old logo TBH

Reply:
It's fine to not have it. I don't care really.